### ISSUE TITLE
Document the environment for kubectl exec

### ISSUE THREAD

**Issue by thockin** [GoogleCloudPlatform/kubernetes#11815]
What PATH is set, etc.  if the PATH is not useful, we should make it useful, so user don't have to 'sh -c'

@ncdc

**Comment by thockin** [comment]
@yujuhong Should this be under CRI spec?

**Comment by yujuhong** [comment]
> @yujuhong Should this be under CRI spec?

That sounds reasonable.

**Comment by paralika** [comment]
Can I work on this?

**Comment by dims** [comment]
Go for it @paralika

**Comment by paralika** [comment]
I looked into it.  It looks like PATH and SHELL values are totally dependent on the container images. I cannot see if we are explicitly passing those while attaching process to TTY.

**Comment by cynepco3hahue** [comment]
@thockin Can you please clarify what exactly should be documented to make it a nice first issue?

**Comment by sftim** [comment]
Is it OK to transfer this to [k/website](https://github.com/kubernetes/website/)?
If so: `/transfer website` will do that (for org members).

**Comment by sftim** [comment]
@thockin what's your view on https://github.com/kubernetes/kubernetes/issues/11815#issuecomment-974686344 ?

**Comment by thockin** [comment]
I feel like it's both "docs" and "API doc" - the latter tends to be in the k/k space.   I don't care where we track this, though.

**Comment by yujuhong** [comment]
I think this is still relevant, but AFAIK no one's actively working on it. 
Tagging a few folks here for visibility since this could be under the CRI spec. @tallclair @samuelkarp @chrishenzie

**Comment by thockin** [comment]
Yeah, I think we still should do this.  I don't think it is very HARD, it just needs someone to do it.

**Comment by cdacosta512** [comment]
So is this really a doc / api-doc change? If so can you point me to where this change needs to go? If it was already posted in this thread forgive me as I was trying to find it. Tks
